### PR TITLE
Init

### PR DESCRIPTION
**Pull request by maxime-guichard**

### CODE REVIEW

**Review by endricklosange — DISMISSED** [review]

**Review by Wowlfy — APPROVED** [review]
oui

File: assets/styles/variables.scss [hunk 1]
```diff
@@ -1,2 +1,10 @@
-$wild-color: #ff6c71;
-$light: #fffbfb;
+@import "~bootstrap/scss/bootstrap";
```
Comment (LucyMerlier) [inline assets/styles/variables.scss:1]: should not import bootstrap here (you already imported it in app.scss)

File: assets/styles/app.scss [hunk 2]
```diff
@@ -1,6 +1,29 @@
 @import 'variables.scss';
+@import "~bootstrap/scss/bootstrap";
```
Comment (LucyMerlier) [inline assets/styles/app.scss:2]: why use double quotes?

File: templates/home/index.html.twig [hunk 3]
```diff
@@ -1,8 +1,7 @@
-{% extends 'base.html.twig' %}
+{% extends 'layout.html.twig' %}
 
 {% block title %}Home{% endblock %}
```
Comment (LucyMerlier) [inline templates/home/index.html.twig:3]: so, your website will be in english?.. but you put a "lang=fr" attribute in your html tag?...

File: templates/layout.html.twig [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{% extends base.html.twig %}
+
+{% block title %}Student-Interim{% endblock %}
+
+{% block navbar %}
+{% include '_navbar.html.twig' %}
```
Comment (LucyMerlier) [inline templates/layout.html.twig:6]: you make an include of a file that does not exist => should delete this line

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

**Review by JuMn88 — APPROVED** [review]

**Review by Wowlfy — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**